CONTRIBUTING.md

```diff
@@ -58,7 +58,7 @@ We're following the [Google Go Code Review](https://code.google.com/p/go-wiki/wi
 + Once ready to land your change, squash your commits. Where n is the number of commits in your branch, run
 `git rebase -i HEAD~n`
 
- and subsequently update your remote (you will have to force the push, `git push -f andybons mybranch`). The pull request will update.
+ and subsequently update your remote (you will have to force the push, `git push -f $USER mybranch`). The pull request will update.
```

Comment:
Hmm, these instructions appear to be inconsistent. This one is assuming that you're pushing to a fork, while the others are using a branch in the origin repo. Personally I prefer using a fork, but for consistency within this file this line should say `git push -f origin $USER/update-readme`